runtime/common/src/migrations/increase_storage_version.rs

```diff
@@ -55,18 +55,6 @@ where
 
 	#[cfg(feature = "try-runtime")]
 	fn pre_upgrade() -> Result<sp_std::vec::Vec<u8>, sp_runtime::DispatchError> {
-		assert_eq!(
-			P::on_chain_storage_version(),
-			FROM_VERSION,
-			"Unexpected onchain version: Expected {FROM_VERSION:?}, received {:?}",
-			P::on_chain_storage_version(),
-		);
-		assert_eq!(
-			P::current_storage_version(),
-			TO_VERSION,
-			"Unexpected upgrade version: Expected {TO_VERSION:?}, latest {:?}",
-			P::on_chain_storage_version(),
-		);
```

Comment:
Maybe I forgot to comment on this. Now, the `try-runtime` binary has new features, such as checking idempotent migrations. Those checks made fail the idempotation check because we're expecting a state that changed in a previous call.

Anyways from now we can use this added migration: https://github.com/paritytech/substrate/pull/14311